src/styles/tailwind-base.css

```diff
@@ -2,6 +2,7 @@
 
 @layer base {
   :root,
+  .light,
```

Comment:
@avidreder I think we want to add back in light and dark classes to correspond to the respective theming. This should help with backwards compatibility for those still using root level html classes for light and dark mode.